### ISSUE TITLE
DOIs are not set for the stable releases

### ISSUE THREAD

**Issue by dimpase** [sagemath/sage#39086]
### Steps To Reproduce

Currently (10.5 was just released) the latest stable release for which DOI is  correctly shown is 10.3.

### Expected Behavior

https://zenodo.org/search?q=sagemath&l=list&p=1&s=10&sort=bestmatch
should show release10.5

It should also be shown on https://doi.org/10.5281/zenodo.8042260 

### Actual Behavior

it does not - it shows 10.5.rc*, it shows 10.5.beta*
https://zenodo.org/search?q=sagemath&l=list&p=1&s=10&sort=bestmatch

https://doi.org/10.5281/zenodo.8042260  gets rewritten to
https://zenodo.org/records/14250808 - which still points to 10.5.rc2!

### Environment

web

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

**Comment by dimpase** [comment]
@vbraun - what do you do when creating 10.*.rc*, but don't do for stable releases, which prevents Zenodo record being created ? (yes, I know, Zenodoo records creation is some sort of dark magic...)

**Comment by vbraun** [comment]
I'm not doing anything different, really. Presumably there is a version search pattern somewhere that goes wrong? :shrug:

**Comment by dimpase** [comment]
The difference is that it's different branches, and I guess they only track the default branch (develop), not master.
@haraldschilly ? Can you take a look? I can't even submit a bug report to zenodo as I don't have the login.

**Comment by dimpase** [comment]
One way would be to just change the default branch to master. Would this cause any problems? (I guess then something should be changed in README.md then)

**Comment by haraldschilly** [comment]
@dimpase I don't remember anything about this. I just checked, and when I open this reference to "SageMath" on https://zenodo.org/records/14600115 I get `Published January 5, 2025 | Version 10.6.beta3`

So, something must have happened in the past month … In my account (personal) there is an enabled integration with github/sagemath/sage and it tells me it will create DOIs automatically if there are any. Could it be that more than one of us has such an integration and this causes an issue?

**Comment by dimpase** [comment]
Perhaps @vbraun could manually re-tag 10.5, and then we'll see if it triggers any action on zenodo?

**Comment by haraldschilly** [comment]
> Are you this user, @haraldschilly ?

Yes, and now I have disabled this. So, **probably important**, before @vbraun re-tags 10.5, someone should enable that integration. 

> Citation metadata load failed

Thank's, I had no idea where to look.

**Comment by dimpase** [comment]
I've re-enabled it - it showed "error 400" first, but then it appeared fine.
